### DISCUSSION TITLE
[Feature Request] Drag and drop device shifting

### DISCUSSION THREAD

**Discussion by Nomad-0** [File-New-Project/EarTrumpet#899] [Ideas]
Currently, we have to go through several processes to shift an application from one device to another. It would be great if it was possible to drag an application from a device to another for instant shifting
![image](https://user-images.githubusercontent.com/76395594/129859996-f0fadbe3-fb50-4c4e-a6bf-c4686ede4db2.png)

**Comment by James-Firth** [comment]
Looks like this is a duplicate of #662, came here to make this request myself :)

**Reply by theiwaz** [reply]
I've tagged other discussions as well with a ref to #662 votes are running up in the 50ies if you add them up 😁

**Comment by theiwaz** [comment]
see #662
